public/Miscellaneous/printYoutube.css

```diff
@@ -35,6 +35,25 @@
 	outline: 0
 }
 
+#qrcode {
+	width:160px;
+	height:160px;
+	margin-top:15px;
+  }
+
+.thisqrcode {
+	padding: 20px; 
+	background-color: white;
+	z-index: 100000; 
+	position:absolute; 
+	top: 0px;
+  }
+  
+div[id^=this_qrcode] {
```

Comment:
Does this line hide all divs that aren't #this_qrcode? This would cause problems if deployed.